frontends/parsers/p4/p4parser.ypp

```diff
@@ -1357,9 +1359,13 @@ expression
     | dotPrefix nonTypeName              { $$ = new IR::PathExpression(new IR::Path(*$2, true)); driver.structure->clearPath(); }
     | expression "[" expression "]"      { $$ = new IR::ArrayIndex(@1 + @4, $1, $3); }
     | expression "[" expression ":" expression "]" { $$ = new IR::Slice(@1 + @6, $1, $3, $5); }
+    | "{" expressionList "," "..." "}"   { $$ = new IR::ListExpression(@1 + @5, *$2, true); }
     | "{" expressionList "}"             { $$ = new IR::ListExpression(@1 + @3, *$2); }
+    | "{" "..." "}"                      { $$ = new IR::ListExpression(@1 + @3, *(new IR::Vector<IR::Expression>()), true); }
     | "{" kvList "}"                     { $$ = new IR::StructExpression(
                                                   @1 + @3, IR::Type::Unknown::get(), (IR::Type_Name*)nullptr, *$2); }  // experimental
+    | "{" kvList "," "..." "}"                     { $$ = new IR::StructExpression(
```

Comment:
I guess this is no longer experimental, that comment (and the previous one) can be removed.